requirements.txt

```diff
@@ -50,7 +50,7 @@ pbr==4.0.4
 petname==2.2
 pexpect==4.6.0
 pickleshare==0.7.4
-pluggy==0.7.1
+pluggy==0.13.0
```

Comment:
you can even remove this and let pytest pick it up as a dependency. None uses it directly.

Reply:
I'll follow up with a proposed change to poetry which has a lockfile included, and just keep this in here for now.